test/unit/google-apis/auth-client.js

```diff
@@ -21,15 +21,13 @@ var proxyquire = require('proxyquire');
 var Configuration = require('../../../src/configuration.js').Configuration;
 
 function verifyReportedMessage(config1, errToReturn, expectedLogs) {
-  class ServiceStub {
-    constructor() {
-      this.authClient = {
-        getToken: function(cb) {
-          cb(errToReturn);
-        },
-      };
-      this.request = function() {};
-    }
+  function ServiceStub() {
```

Comment:
Why does `ServiceStub` have to change from a class to a function?

Reply:
The name `ServiceStub` is referenced below and is used as a function.  If a `class` is used, the unit tests fail when `ServiceStub` is used with an error stating that `new` must be used when constructing the object.